src/jemalloc.c

```diff
@@ -154,6 +154,8 @@ fxp_t		opt_narenas_ratio = FXP_INIT_INT(4);
 
 unsigned	ncpus;
 
+unsigned	opt_debug_dblfree_cbin_scan_limit;
```

Comment:
For setting the default value, here you can do `unsigned opt_debug_double_free_max_scan = SAFETY_CHECK_DOUBLE_FREE_MAX_SCAN_DEFAULT;`
and then in `safety_check.h` have
```
#define SAFETY_CHECK_DOUBLE_FREE_MAX_SCAN_DEFAULT 32
```

Also let's add this in https://github.com/jemalloc/jemalloc/blob/dev/src/jemalloc.c#L1734
```
if (!config_debug) {
    opt_debug_double_free_max_scan = 0;
}
```

It doesn't really affect anything, but should make the stats output clearer (that it's disabled).

Reply:
addressed in [5a2502a](https://github.com/jemalloc/jemalloc/pull/2310/commits/5a2502a2dd9fb3e401373e087983f334ed35dbc4)